internal/watcher/instance/instance_watcher_service.go

```diff
@@ -224,6 +199,26 @@ func (iw *InstanceWatcherService) checkForUpdates(
 	}
 }
 
+func (iw *InstanceWatcherService) sendNginxConfigContextUpdate(
+	ctx context.Context,
+	nginxConfigContext *model.NginxConfigContext,
+) {
+	if iw.nginxConfigCache[nginxConfigContext.InstanceID] == nil ||
+		!iw.nginxConfigCache[nginxConfigContext.InstanceID].Equal(nginxConfigContext) {
```

Comment:
Tiny style suggestion, same effect though.
```suggestion
	if cachedCtx, ok := iw.nginxConfigCache[nginxConfigContext.InstanceID]; ok && !cachedCtx.Equal(nginxConfigContext) {
```